DABI.js

```diff
@@ -0,0 +1,677 @@
+    var namespace = doc.documentElement.namespaceURI;
+    var nsResolver = namespace ? function(prefix) {
+        if (prefix == 'x') return namespace;
+        else return null;
+    } : null;
+
+    //variable declarations
+    var newItem = new Zotero.Item('journalArticle');
+    newItem.url = doc.location.href;
+    newItem.title = "No Title Found";
+
+    var items = new Object();
+    var headers;
+    var contents;
+    var link;
+    var blankCell = "temp";
+    var headersTemp;
+    var tagsContent = new Array();
+
+    var myXPathObject = doc.evaluate('//tr/th', doc, nsResolver, XPathResult.ANY_TYPE, null);
+    var myXPathObject2 = doc.evaluate('//tr/td', doc, nsResolver, XPathResult.ANY_TYPE, null);
+    var link = doc.evaluate('//tr/td/a', doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext();
+
+    //While Loop to populate "items" Object and save tags to an Array.
+    while (headers = myXPathObject.iterateNext()) {
+
+        headersTemp = headers.textContent;
+        if (!headersTemp.match(/\w/)) {
+            headersTemp = blankCell;
+            blankCell = blankCell + "1";
+        }
+
+        contents = myXPathObject2.iterateNext().textContent;
+
+        if (headersTemp.match("temp")) {
+            tagsContent.push(contents);
+        }
+
+        items[headersTemp.replace(/\s+/g, '')] = contents.replace(/^\s*|\s*$/g, '');
+
+    }
+
+    //Formatting and saving "title" fields
+    if (items["Titel"]) {
+        newItem.title = items["Titel"].replace(/\*/g, '');
+        var short = newItem.title.replace(/^(Die )|(Der )|(Das )/, '');
+        short = short.replace(/(?:\S+\s+){2}\S+/);
+        newItem.shortTitle = short.substring(0, 1).toUpperCase() + short.slice(1);
+    }
+
+    if (items["Untertitel"]) {
+        newItem.title = newItem.title.concat(": ", items["Untertitel"]);
+    }
+
+    //Formatting and saving "Author" field
+    if (items["Autoren"]) {
+        var author = items["Autoren"];
+        if (author.match("; ")) {
+            var authors = author.split("; ");
+            for (var i in authors) {
+                newItem.creators.push(Zotero.Utilities.cleanAuthor(authors[i], "author", true));
+            }
+        } else {
+            newItem.creators.push(Zotero.Utilities.cleanAuthor(author, "author", true));
+        }
+    }
+
+    //Formatting and saving "pages" field
+    if (items["Anfangsseite"] && items["Endseite"]) {
+        newItem.pages = items["Anfangsseite"].concat("-", items["Endseite"]);
+    } else {
+        newItem.pages = items["Anfangsseite"];
+    }
+
+    //Saving the tags to Zotero
+    if (items["Schlagwörter"]) {
+        var tags = items["Schlagwörter"];
+        if (tags.match("; ")) {
+            var tagsContent = tags.split("; ");
+        } else {
+            newItem.tags.push(tagsContent);
+        }
+        for (var i in tagsContent) {
+            newItem.tags.push(tagsContent[i]);
+        }
+    }
+
+    //Link
+    newItem.url = link;
```

Comment:
What is this URL pointing to? I don't see any links in the table on the sample page I am looking at. (also, you should move the relevant code to the same location. You've already assigned `newItem.url` to something else above and the assignment to link is also way above)